docs/content/troubleshooting/log_status_collect.md

```diff
@@ -1,41 +1,121 @@
-# Log and status collection
+# Detecting errors and log collection
 
 Use the CSI (Container Storage Interface) driver debug information for problem identification.
 
 **Note:** These procedures are applicable for both Kubernetes and Red Hat® OpenShift®. For Red Hat OpenShift, replace `kubectl` with `oc` in all relevant commands.
 
-To collect and display status and logs related to the different components of IBM® block storage CSI driver, use the following Kubernetes commands and procedures:
+## Detecting errors
+
+To help pinpoint potential causes for stateful pod failure:
+
+1.  Verify that all CSI pods are running.
+    ```
+    kubectl get pods -n <namespace> -l csi
+    ```
+
+2.  If a pod is not in a _Running_ state, run the following command:
+    ```
+    kubectl describe -n <namespace> pod/<pod-name>
+    ```
+    View the logs.
+
+## Status and log collection
+To collect and display status and logs related to the different components of IBM® block storage CSI driver, use the Kubernetes commands that are found in this section.
+
+Before you begin collecting logs, create a directory for the logs on the control-plane or bastion server.
+
+For example:
+
+```
+mkdir logs
+```
+Save logs and status reports directly to the created directory by adding in the following string at the end of collection command: `> logs/<log_filename>`.
+
+**Important:** Be sure that the logs cover any relevant timeframes for the specific issues that you are trying to debug when gathering logs from the storage system.
+
+### General status and log collection for IBM Support
+Be sure to run the following steps and copy the output to an external file, when engaging IBM Support and sending log collections.
+
+1. Check the node status.
+    
+    `kubectl get nodes`
+2. Check the CSI driver component status.
+
+    `kubectl get -n <namespace> pod -o wide | grep ibm-block-csi`
```

Comment:
why is it sometimes `<namespace>` and sometimes `--all-namespaces`?
IMO `--all-namespaces` makes us less blind and easier for the user to run

Reply:
The only place it is --all-namespaces is for 

> ### Log collection for CSI operator logs
> To collect CSI operator logs, use the following commands:
> 
>     operatorpod=`kubectl get pods --all-namespaces |grep ibm-block-csi-operator|awk '{print $2}'`
>     kubectl logs $operatorpod -n <namespace> > logs/operator

@roysahar  can this be updated to <namespace>?

Reply:
@roysahar ? can this be changed?

Reply:
updated in other place with grep command

Reply:
done